### PR TITLE
Update README file to reflect the current implementation

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -9,29 +9,42 @@ environment - given the list of package that should be installed as well as
...
+
+1. Assemble/build the given software by installing requested native packages
+   and/or Python packages into a container image. OpenShift's ImageStreams and
+   builds are used under the hood.
+
+2. Execute the given application using a script that is provided by a user -
+   there are run "inspection jobs" that execute provided user script. All the
+   information related to the node where the inspection job was run are
+   aggregated.  Such information consists of hardware available (such as CPU,
+   CPU flags and features, and such; see `mun-hwinfo
```
Comment (pacospace) [inline README.rst:25]: `amun-hwinfo` missing a

**Review by pacospace — COMMENTED** [review]
One minor comment

File: README.rst (hunk 1 above)
Comment (fridex) [reply README.rst]: Nice catch. Updated :+1:

**Review by pacospace — APPROVED** [review]
Very nice!! Thanks!

**Pull request merged**